yoeo/utils/loss.py

```diff
@@ -152,7 +152,20 @@ def build_targets(p, targets, model):
         t = targets * gain
         # Check if we have targets
         if nt:
-            t = t.squeeze(0)
+            # Check if we use anchor boxes
+            if na == 1:
```

Comment:
In both cases, you are doing the `t = t.squeeze(0)`, so you could simplify it.

Reply:
You are right